src/key.h

```diff
@@ -6,6 +6,7 @@
 #ifndef BITCOIN_KEY_H
 #define BITCOIN_KEY_H
 
+#include "pubkey.h"
```

Comment:
From my POV, that's on the list of "safe to include everywhere" since it's used in libbitcoinconsensus.